src/Smalot/PdfParser/Document.php

```diff
@@ -144,9 +149,119 @@ protected function buildDetails()
             $details['Pages'] = 0;
         }
 
+        $details = array_merge($details, $this->metadata);
+
         $this->details = $details;
     }
 
+    /**
+     * Get XMP Metadata
+     */
+    public function getXMPMetadata(string $content)
```

Comment:
A get function always returns something in my opinion, but this function extracts meta data. Therefore I propose renaming it to `extractXMPMetadata` instead.